components/Navigation/SectionNavigation.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from "next/link";
+import sections from "./sections/sections.json";
+
+const SectionNavigation = () => {
+  return (
+    <ul>
+      {sections.map(({ title, href }) => (
+        <li key={href}>
+          <Link href={href}>
+            <a>{title}</a>
+          </Link>
+        </li>
+      ))}
+    </ul>
+  );
+};
```

Comment:
Extra return statement
```suggestion
const SectionNavigation = () => (
  <ul>
    {sections.map(({ title, href }) => (
      <li key={href}>
        <Link href={href}>
          <a>{title}</a>
        </Link>
      </li>
    ))}
  </ul>
);
```